distconf/centos-6-x86_64.yaml

```diff
@@ -0,0 +1,8 @@
+extends: "lib/centos.yaml"
+
+os:
+  version: 6
+  arch: x86_64
+
+docker:
+  from: centos:centos6
```

Comment:
registry.centos.org or docker.io, votes?

Reply:
This value is now in all sclorg's repositories set to centos:centos6, we might consider adding configurations on different base image registries in the future.